pennylane_lightning_gpu/lightning_gpu.py

```diff
@@ -389,9 +381,6 @@ def var(self, observable, shot_range=None, bin_size=None):
 
         return squared_mean - (mean**2)
 
-    def sample(self, **kwargs):
```

Comment:
Hi @mlxd. I suppose this no longer is required. right?

Reply:
@trevor-vincent  I'll defer to you?

Reply:
Yep not required!